torch/csrc/jit/pickler.h

```diff
@@ -152,6 +152,9 @@ class Pickler {
   void pushTuple(const IValue& ivalue);
   void pushString(const std::string& string);
   void pushDevice(const IValue& ivalue);
+  #ifdef USE_DISTRIBUTED
+    void pushRRef(const IValue& ivalue);
+  #endif
```

Comment:
Why do we need ifdef here? This is just a function declaration.

Reply:
same as above